### DISCUSSION TITLE
Target temperature jumping to high values without input

### DISCUSSION THREAD

**Discussion by mbrucher** [jmcollin78/versatile_thermostat#583] [Q&A] [answered]
## Version of the custom_component
6.3.0

## Configuration
I don't know where this can be retrieved frim the UI without trying to find the actual files? 
It's a thermostat on top of an existing climate (moes 002)

The issue is simple, for whatever reason, the target temperature is modified by something. I've seen the same with 006 thermostats from moes as well.
The temperature could have come from the appliance itself, but some values are gigantic (several millions) and switch in the middle of the night, which doesn't make sense me (the appliance doesn't have something switching in the middle of the night and not with these values, and there is no schedule active on the thermostat at that time.

**Comment by federkicco** [comment]
+1  facing the same issue with the Moes BHT-002 thermostats.

I installed them just few days ago, on version 6.3.0, so I don´t have a way to know if this worked before.
On this same setup I have some ESPhome based thermostats too, and those are not exhibiting this issue.
I'm using the scheduler component to change the preset during the day, but these jumps have happened at times when there was no scheduled settings change.

For the time being, as a workaround, I'm using an automation to reset the status when the set point goes outside a given range (4-30 degrees Celsius).

I'm going to check the HA and Zigbee2MQTT logs this weekend.. I'll report if I find something.

**Comment by mbrucher** [comment, marked as answer]
Oh thank you for confirming!
Can you share the automations you created to reset the status?

I think it's a bug in the MOES side, they do report target temp jumping, I think I rememeber seeing a graph that was very off.

**Comment by mbrucher** [comment]
Yeah, I can confirm it's the 002, the 006 doesn't have the problem, so I'm just going to replace the 2 ones I have with the better 006.
